### PR TITLE
tweak TTL/Hop Limit for ownership messages

### PR DESCRIPTION
**Pull request by boucadair**
Add a provision for the control of the forwarding scope of the ownership message. The goal is to avoid overloading servers.

### CODE REVIEW

File: draft-wing-cidfi.md [hunk 1]
```diff
@@ -470,11 +470,13 @@ CIDFI-aware network elements, service-cidfi and wi-fi.
    "cidfi":[
       {
          "cidfinode":"service-cidfi.example.net",
+         "min-ttl":3,
```
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-wing-cidfi.md:473]: Thanks, Tiru. Done.

**Review by tireddy2 — APPROVED** [review]

File: draft-wing-cidfi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-wing-cidfi.md:473]: created a separate PR for the new sub registry: https://github.com/danwing/cidfi/pull/62

[the remark this answers is not in the document]
Comment (boucadair) [reply draft-wing-cidfi.md:473]: Agree. Done.

File: draft-wing-cidfi.md [hunk 2]
```diff
@@ -1234,6 +1234,22 @@ Register new special-use domain name cidfi.arpa for DNS SVCB discovery.
 This document requests IANA to register the new DNS SVCB "_cidfi-aware" in
 the "DNS Service Bindings (SVCB)" registry available at {{IANA-SVCB}}.
 
+The document also requests IANA to register the following service parameter
```
Comment (tireddy2) [inline draft-wing-cidfi.md:1237]: We will also have to add cidfipathauth and cidfimetadata.

Comment (boucadair) [reply draft-wing-cidfi.md:1237]: We will need to consider the sync the attributes between the various registries but we also need to make sure that we are not overloading SVCB. Will create an issue for this + not implement more changes as part of this PR.

**Review by danwing — APPROVED** [review]

**Pull request merged**